meta-lmp-base/recipes-samples/images/lmp-image-common.inc

```diff
@@ -36,6 +36,7 @@ CORE_IMAGE_BASE_INSTALL += " \
     sudo \
     zram \
     i2c-tools \
+    dtc \
```

Comment:
Is the compile tool needed for this?

Reply:
Not sure which compile tool you're referring to.  This tool is actually used to compile and decompile the device-tree binaries.